### PR TITLE
Add PyCharm PY and PY SDK support

### PR DESCRIPTION
**Pull request by ARostovsky**

### CODE REVIEW

File: examples/pycharm-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Simple plugin for CLion
```
Comment (zolotov) [inline examples/pycharm-plugin/README.md]: for pycharm

File: README.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ The following attributes are apart of the Patching DSL <kbd>patchPluginXml { ...
 | <kbd>destinationDir</kbd> is a directory to store patched xml files.                  | <kbd>'${project.buildDir}/patchedPluginXmlFiles'</kbd> |
 
 ### Publishing DSL
-The following attributes are a part of the Publishing DSL <kbd>publishPlugin { ... }</kbd> in which allows Gradle to upload a working plugin to the JetBrains Plugin Repository. Note that you need to upload the plugin to the repository at least once manually (to specify options like the license, repository URL etc.) before uploads through Gradle can be used.
```
Comment (zolotov) [inline README.md]: lost changes

File: examples/pycharm-plugin/build.gradle [hunk 3]
```diff
@@ -0,0 +1,5 @@
+intellij {
+    // PC-Snapshot is also available here
```
Comment (zolotov) [inline examples/pycharm-plugin/build.gradle]: really? do we publish community snapshots publicly?

File: README.md (hunk 2 above)
Comment (zolotov) [reply README.md]: Still here?

File: src/main/groovy/org/jetbrains/intellij/dependency/IdeaDependencyManager.groovy [hunk 4]
```diff
@@ -42,7 +42,10 @@ class IdeaDependencyManager {
         } else if (type == 'CL') {
             dependencyGroup = 'com.jetbrains.intellij.clion'
             dependencyName = 'clion'
-        } else if (type == 'RD') {
+        } else if (type == 'PY' || type == 'PC') {
+            dependencyGroup = 'com.jetbrains.intellij.pycharm'
+            dependencyName = 'pycharm' + type
+        }else if (type == 'RD') {
```
Comment (zolotov) [inline src/main/groovy/org/jetbrains/intellij/dependency/IdeaDependencyManager.groovy]: Reformat

File: src/main/groovy/org/jetbrains/intellij/IntelliJPluginExtension.groovy [hunk 5]
```diff
@@ -43,7 +43,11 @@ class IntelliJPluginExtension {
             return "JPS"
         } else if (version.startsWith('CL-') || 'CL' == type) {
             return 'CL'
-        } else if (version.startsWith('RD-') || 'RD' == type) {
+        } else if (version.startsWith('PY-') || 'PY' == type) {
+            return 'PY'
+        } else if (version.startsWith('PC-') || 'PC' == type) {
+            return 'PC'
+        }else if (version.startsWith('RD-') || 'RD' == type) {
```
Comment (zolotov) [inline src/main/groovy/org/jetbrains/intellij/IntelliJPluginExtension.groovy]: Reformat

Comment (ARostovsky) [reply src/main/groovy/org/jetbrains/intellij/IntelliJPluginExtension.groovy]: processed

File: src/main/groovy/org/jetbrains/intellij/dependency/IdeaDependencyManager.groovy (hunk 4 above)
Comment (ARostovsky) [reply src/main/groovy/org/jetbrains/intellij/dependency/IdeaDependencyManager.groovy]: processed

File: examples/pycharm-plugin/build.gradle (hunk 3 above)
Comment (ARostovsky) [reply examples/pycharm-plugin/build.gradle]: yeap, we want to publish pycharm community as well

File: examples/pycharm-plugin/README.md (hunk 1 above)
Comment (ARostovsky) [reply examples/pycharm-plugin/README.md]: processed

**Pull request merged**